docs/modules/ROOT/pages/how-tos/synthesize.adoc

```diff
@@ -0,0 +1,247 @@
+= Synthesize a cluster
+
+:projectsyn-ig: https://vshn.chat/channel/projectsyn-ig[Project Syn IG]
+:portal-lieutenant-tenants: https://control.vshn.net/syn/lieutenanttenants/vshn-lieutenant-prod[control.vshn.net]
+
+TIP: "Synthesize" is the act of making a cluster managed by Project Syn
+
+This how-to will show you how to synthesize a cluster.
+
+== Prerequisites
+
+* `cluster-admin` permissions on the cluster to synthesize.
+* Write access to the Project Syn view on https://control.vshn.net[control.vshn.net]
+
+== Procedure
+
+=== Registering a tenant
```

Comment:
The intention is to treat the word "Tenant" as an object name and therefore capitalize it.

Reply:
If we want to do this, we'll need to add "Tenant" and "Cluster" to the glossary and always have them capitalized (titles, text, ...). Please note that for me it just feels odd (and I mentally stumble while reading) to see the regular English words "tenant" and "cluster" capitalized in a sentence.

Additionally, if we want to standardize on capitalizing "Tenant" and "Cluster", we'll have to go over the entirety of the Project Syn docs, since we're almost certainly not consistent with the capitalization right now. As far as I remember, all the documentation I've written doesn't consciously capitalize "tenant" and "cluster".

Reply:
As I have a low LOGAF here, let's just not do anything about it for now.